webpack.config.js

```diff
@@ -153,6 +159,17 @@ const wordpressExternals = ( context, request, callback ) =>
 		? callback( null, `root ${ wordpressRequire( request ) }` )
 		: callback();
 
+function chooseTerserEcmaVersion( supportedBrowsers ) {
+	if ( ! caniuse.isSupported( 'arrow-functions', supportedBrowsers ) ) {
+		return 5;
+	}
+	if ( ! caniuse.isSupported( 'es6-class', supportedBrowsers ) ) {
+		return 5;
+	}
+
+	return 6;
```

Comment:
could this be higher? would it matter?

Reply:
It could, but looking at the codebase for Terser at the moment, none of the checks deal with any version > 6. It may make sense to keep an eye on this to reap any potential future benefits, but for now I'd say it's best to keep things simple.